pyproject.toml

```diff
@@ -41,7 +41,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette>=0.36.3,<0.37.0",
+    "starlette>=0.31.7",
```

Comment:
I think this wants to be 0.37.1, rather than 0.31.7

Reply:
Right you are thanks!